debian/mariadb-server.postinst

```diff
@@ -242,6 +242,23 @@ EOF
     chown 0:0 $dc
     chmod 0600 $dc
 
+    feedback_cnf=$mariadb_cfgdir/mariadb.conf.d/feedback.cnf
+    if [ ! -f $feedback_cnf ]; then
+      if db_input high mariadb-server/feedback_optin; then
+        db_go || true
+      fi
+      db_get mariadb-server/feedback_optin
+      {
+        echo "# THIS IS A GENERATED FILE. ALL CHANGES TO IT MIGHT BE LOST.";
+        echo "[server]";
+        if [ "$RET" = true ]; then
+          echo "enable-feedback";
```

Comment:
https://mariadb.com/kb/en/feedback-plugin/#enabling-the-plugin saysa:

```
[mariadb]
...
feedback=ON
```

Please choose what you consider the canonical way and use it everywhere to avoid confusion by having different settings in different places to do the same thing.